testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1995,6 +1995,28 @@ QUnit.module('events', moduleSetup, () => {
         assert.strictEqual(actionData.itemIndex.group, 1, 'correct groupIndex passed');
     });
 
+    QUnit.test('onClick for item should be fired on \'enter\' key pressed', function(assert) {
+        if(!isDeviceDesktop(assert)) {
+            return;
+        }
+
+        const $element = this.element.dxList({
+            items:[{
+                text: 'item 1',
+                onClick: () => {
+                    assert.ok('onClick was fired');
+                }
+              }
+            ]
+        });
+
+        const $item = $element.find(toSelector(LIST_ITEM_CLASS));
+
+        $item.trigger('dxpointerdown');
```

Comment:
why do we do this? For focusing? If yes, we can write it more intuitive way
```suggestion
        $item.focus();
```

Reply:
Yes for focusing, but $item.focus() doesn't work, may be dxpointerdown used in all other places for a reason